tasks/add-unique-granuleID/README.md

```diff
@@ -0,0 +1,106 @@
+# @cumulus/add-unique-granuleID
+
+This is a [Cumulus](https://nasa.github.io/cumulus) task which takes the following actions on each granule in an incoming set of payload granules:
+
+- Adds the existing `granuleId` to the `producerGranuleId` key
+- Updates the existing `granuleId` field to a 'unique' value based on the algorithim used in @cumulus/ingest `generateUniqueGranuleId`
+
+## Usage
+
+This lambda takes the following input and config objects, derived from workflow configuration using the [Cumulus Message Adapter](https://github.com/nasa/cumulus-message-adapter/blob/master/CONTRACT.md) to drive configuration from the full cumulus message:
+
+### Input
+
+The input takes a list of 'pre-sync' or Cumulus API formatted granules, with the following required fields:
```

Comment:
I don't think pre-sync is a term we're using elsewhere. We may want to be explicit here. We can probably just say Cumulus API formatted granules.

Reply:
They're not Cumulus API formatted granules though.     The schema for discover granules/pdrs/etc that matches the input schema for sync granules is *not* the same as the API granule.

Reply:
I updated this to remove the custom term (69fa8f07be5), which leaves the doc explicitly just defining the required fields - I'm not sure this won't lead to other confusion, but that's more a function of that difference *being* confusing than the changes here.

Reply:
Yeah, that's fine. Agreed that the issue here isn't this documentation, it's that we have a few different forms a Granule can take and we don't really have canonical names.